overlord/devicestate/devicestate_test.go

```diff
@@ -1138,15 +1138,49 @@ func (s *deviceMgrSuite) TestDeviceManagerEmptyOperatingModeRun(c *C) {
 	c.Check(s.mgr.OperatingMode(), Equals, "run")
 }
 
-func (s *deviceMgrSuite) TestStartOfOperationTimeFromSeedTime(c *C) {
+type startOfOperationTimeSuite struct {
+	state  *state.State
+	mgr    *devicestate.DeviceManager
+	runner *state.TaskRunner
+}
+
+var _ = Suite(&startOfOperationTimeSuite{})
+
+func (s *startOfOperationTimeSuite) SetUpTest(c *C) {
+	dirs.SetRootDir(c.MkDir())
+	os.MkdirAll(dirs.SnapRunDir, 0755)
+
+	s.state = state.New(nil)
+	s.runner = state.NewTaskRunner(s.state)
+	s.mgr = nil
+}
+
+func (s *startOfOperationTimeSuite) TearDownTest(c *C) {
+	dirs.SetRootDir("")
+}
+
+func (s *startOfOperationTimeSuite) manager(c *C) *devicestate.DeviceManager {
+	if s.mgr == nil {
+		hookMgr, err := hookstate.Manager(s.state, s.runner)
```

Comment:
(nitpick⁹⁹) it looks like all tests in the startOfOperationTimeSuite use s.mgr we could always do this in SetUpTest I think. But not a blocker or even worth changing, just an observation.